src/transformStateWithClones.js

```diff
@@ -6,8 +6,47 @@
  *
  * @return {Object[]}
  */
+
 function transformStateWithClones(state, actions) {
-  // write code here
+  const result = [];
+
+  const copy = { ...state };
+
+  for (const action of actions) {
+    if (action.type === 'addProperties') {
```

Comment:
'if' seems must be implement through 'switch case' (In [checklist](https://github.com/mate-academy/js_stateful-clones/blob/master/checklist.md) it's also includes 👀)
I'm like you, don't adapted for this things 😅

Reply:
Good poin!